### ISSUE TITLE
AppImage missing for 1.0.01

### ISSUE THREAD

**Issue by probonopd** [OtterBrowser/otter-browser#1559]
The AppImage is missing for 1.0.01:
https://sourceforge.net/projects/otter-browser/files/otter-browser-1.0.01/

**Comment by crass** [comment]
Have the AppImage builds been discontinued due to #1233?  It would be nice to get a newer AppImage with out the SSL issue.

**Comment by probonopd** [comment]
Is this something appimage-builder could take care of @azubieta?

**Comment by azubieta** [comment]
> Is this something appimage-builder could take care of @azubieta?

Yes, appimage-builder bundles libssl which solves the issue. But you must a Debian/Ubuntu system with the latest libssl updates, to avoid known security issues.

**Comment by danger89** [comment]
Indeed. I'm still missing a appimage.
